### PR TITLE
The Ammolathe can no longer print NT12 and Automag magazines.

### PR DESCRIPTION
**Pull request by Irradiation**
It's okay when I decry ~~griff~~ FUN that pertains to other departments, but not when others do it to my departments.

On a slightly more serious note, I should have paid more attention to the Ammolathe PR, because good lord this is a case of Vectors being borked and the c*der behind it powercreeping his content intentionally. For the record, you can currently take magazines meant for guns that are intended to be adminbus/wizard/traitor-only (NT12 and Automag) and load them into a Vector. Said vector just so happens to be semi-auto and have a really nice burst-fire feature. You know what regular shotguns don't have? Semi-auto and burst-fire. I lurv me some 120 brute in half a second (quoting Sonix here). The only shotgun that doesn't quite get rendered obsolete by the Vector is the Combat Shotgun, because of its 8-round capacity.

Why the Auytomag and NT12 more specifically as opposed to the other magazines? That's simple. They both use calibers that are readily available to the crew/the people that would handle a Vector (Security has boxes of shotgun ammo and can print more at the ammolathe, .357 can be printed at an autolathe). Why not the other magazines? Because outside of 9mm, they're practically unobtainium calibers on the station.

tl;dr Vectors are borked and Ammolathe highlighted their borkedness even more.

[balance][tweak]

:cl:
 * rscdel: The ammolathe can no longer print NT12 and Automag magazines.

### CODE REVIEW

**Comment by SonixApache** [comment]
👍 
could do without the bitching in the OP

**Comment by Irradiation** [comment]
Discord drama is best drama

**Comment by aceedex** [comment]
That practically makes vectors shitty glocks. Which is WHY tier
instead, remove burst fire feature. 
Ballistics are still shit, by the way

**Comment by bathosbathtime** [comment]
>1 hour
>
why so fast

**Comment by MadmanMartian** [comment]
because this was something snuck in by DrSlimes. Not happy with him nor it.

**Comment by DrSnips** [comment]
I clearly laid everything out in the OP. Don't spin it as something I snuck something in.

**Comment by SonixApache** [comment]
@MadmanMartian follow the guidelines for collabs

**Comment by Irradiation** [comment]
Snuck is or not, shit be broke

**Comment by Hoshi-chan** [comment]
What the fuck, did you really just remove the only viable reason to use the vector over literally anything else in the armory? No emojis? No server vote?
